src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -23,7 +23,7 @@ const createNewStep = (context, step, args) => {
     Type: 'Task',
     Resource: activityArn,
     ResultPath: null,
-    TimeoutSeconds: 300,
+    TimeoutSeconds: 600,
```

Comment:
Juanlu reported that it took near to 20 minutes to complete de integration for that dataset, perhaps we could increase it a bit more?